tests/console-dot/basic.spec.ts

```diff
@@ -1,26 +0,0 @@
-import { test, expect } from '@playwright/test';
```

Comment:
Shouldn't we leave also basic test for login and logout function ? IMHO we always want to test this. Or at least leave logout test to verify function of deleting session.

Reply:
Isn't it use-case of consoledot instead of rhosak? I can revert changes related to the deletion of the tests if it is relevant, but all test cases will fail in case of login fails. And logout is used in the tests for now so it is covered from my POV.